Makefile

```diff
@@ -44,6 +44,7 @@ default:
 	@echo "  docs             - generate Sphinx HTML documentation, including API docs"
 	@echo "  buildjs          - generate js static assets"
 	@echo "  buildcss         - generate css static assets"
+	@echo "  download         - re-download vendor source and test data"
```

Comment:
While this is downloading things, I think the point of it is more helpful for a make target. It's updating vendored things that age over time. I wonder if there's a better name for this make target. Maybe "update-vendored" or something like that?